packages/dara-core/js/shared/dynamic-component/dynamic-component.tsx

```diff
@@ -321,6 +321,13 @@ function PythonWrapper(props: PythonWrapperProps): JSX.Element {
     }
 
     if (isRawString(component)) {
+        let res = component.props.content
+        if(res.startsWith('__dara__')){
+            res = res.slice(8);
+            const resJson = JSON.parse(res);
+            console.log(resJson);
+            return <DynamicComponent component={resJson} key={component.uid} />;
+        }
```

Comment:
I'm not sure this logic belongs here, can this be handled on the app side? Happy to discuss privately